control

```diff
@@ -695,15 +695,16 @@ class StepInit(Context, collections.Callable):
         return [subthing for subthing in subthings
                 if subthing in subtest_modules]
 
-# Grab a compressed copy of the audit log after every test
+# Grab a compressed copy of the audit log and messages after every test
 # N/B: Autotest silently ignores any failures with this.
-audit_log = '/var/log/audit/audit.log'
-if os.path.isfile(audit_log):
-    command = 'gzip -9c %s' % audit_log
-    # on_every_test=False has already run (before control file loaded)
-    job.add_sysinfo_command(command, "audit.log.gz", on_every_test=True)
-else:
-    logging.warning("Can't record %s because it doesn't exist", audit_log)
+for gzname, filepath in {'audit.log.gz': '/var/log/audit/audit.log',
+                         'messages.gz': '/var/log/messages'}:
```

Comment:
Why the duplication? Why not just enumerate paths, and use `os.path.basename(filepath)` instead of gzname?